<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>statistics</class>
 <widget class="QDialog" name="statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="lb_player">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QPushButton" name="bt_close">
     <property name="text">
      <string>close</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="bt_next">
     <property name="text">
      <string>&gt;&gt;</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="bt_prev">
     <property name="text">
      <string>&lt;&lt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QLabel" name="lb_t1">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="lb_t2">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="lb_t3">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="lb_rimb">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="lb_steal_to">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLabel" name="lb_fouls">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="2">
    <widget class="QLabel" name="lb_stopp">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="lb_asst">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lb_points">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="lb_val">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLabel" name="lb_min">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="lb_plusminus">
     <property name="text">
      <string>-</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
